poem/src/web/static_file.rs

```diff
@@ -167,17 +169,23 @@ impl StaticFileRequest {
             etag: None,
             last_modified: None,
             content_range,
+            cache_control: None,
         })
     }
 
     /// Create static file response.
     ///
-    /// `prefer_utf8` - Specifies whether text responses should signal a UTF-8
-    /// encoding.
+    /// # Arguments
+    ///
+    /// * `prefer_utf8` - Specifies whether text responses should signal a UTF-8
+    ///   encoding.
+    /// * `no_cache` - Specifies whether to set the `Cache-Control` header to
+    ///   `no-cache`.
     pub fn create_response(
```

Comment:
might be that it would be cleaner with and options bag rather than two bool arguments. didn't think about the fact this was a breaking change to begin with 😅